src/runtime/Value.h

```diff
@@ -369,7 +369,25 @@ class Value {
     Type m_type;
 };
 
-inline size_t valueSizeInStack(Value::Type type)
+inline size_t valueSize(Value::Type type)
+{
+    switch (type) {
+    case Value::I32:
+        return sizeof(int32_t);
+    case Value::F32:
+        return sizeof(float);
+    case Value::I64:
+        return sizeof(int64_t);
+    case Value::F64:
+        return sizeof(double);
+    case Value::V128:
+        return 16;
+    default:
+        return sizeof(size_t);
```

Comment:
What about this default case?
Is that any possible case that `type` could be another form?

Reply:
No. I would add assert not reached